pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -359,6 +359,21 @@ private void registerServiceStatusHandler() {
         new ServiceStatus.MultipleCallbackServiceStatusCallback(serviceStatusCallbackListBuilder.build()));
   }
 
+  private Set<String> getConsumingSegments(String realtimeTableName) {
```

Comment:
(minor) Annotate return as `@Nullable`

Reply:
thought we mainly annotate public methods. but this method is leaked to other places as lambda, so will annotate.